libcloud/compute/base.py

```diff
@@ -997,9 +997,9 @@ def create_volume(self, size, name, location=None, snapshot=None):
                                (optional)
         :type location: :class:`.NodeLocation`
 
-        :param snapshot:  Name of snapshot from which to create the new
-                               volume.  (optional)
-        :type snapshot:  ``str``
+        :param snapshot:  Snapshot from which to create the new
```

Comment:
This is a breaking change - can you please document it in docs/upgrade_notes.rst?